common-ui/src/main/java/com/duckduckgo/mobile/android/ui/view/OutLinedTextInput.kt

```diff
@@ -154,8 +162,6 @@ class OutLinedTextInputView @JvmOverloads constructor(
     private fun setupPasswordMode() {
         binding.internalPasswordIcon.visibility = VISIBLE
         binding.internalEditText.inputType = EditorInfo.TYPE_TEXT_VARIATION_PASSWORD or EditorInfo.TYPE_TEXT_FLAG_MULTI_LINE
-        // This has to be set before the transformationMethod. Else, it will force the password to be visible.
```

Comment:
removing this forces the password to be single line even if we set the inputType just above to EditorInfo.TYPE_TEXT_FLAG_MULTI_LINE. (You'll see that if you open the edit mode in password it won't expand anymore). If that is the desired behavior, then i think we can remove ditorInfo.TYPE_TEXT_FLAG_MULTI_LINE in line 156.